src/clusterfuzz/_internal/bot/tasks/commons/testcase_utils.py

```diff
@@ -0,0 +1,68 @@
+# Copyright 2024 Google LLC
```

Comment:
I've only seen directories called "common" not "commons".

Reply:
The weirdness is more because something in _internal/cron is using something in _internal/bot. Maybe this should belong in _internal/common